profile/README.md

```diff
@@ -1,4 +1,4 @@
-![OpenSearch Banner](banner.jpg)
+![OpenSearch logo and name on top of a dark blue background with a slight honeycomb patttern](https://raw.githubusercontent.com/opensearch-project/.github/main/profile/banner.jpg)
```

Comment:
nice, but did you mean to misspell patttern? :)

Reply:
You know I read it like three times and missed that. Update inbound in a few

Reply:
@dblock Resolved spelling issue.